scripts/RocketHideoutElevator.asm

```diff
@@ -80,6 +80,6 @@ RocketHideoutElevatorText1:
 	jp TextScriptEnd
 
 RocketHideoutElevatorText_4578b:
-	TX_FAR _RocketElevatorText_4578b
-	TX_WAIT
-	db "@"
+	text_far _RocketElevatorText_4578b
+	text_linkpromptbutton
```

Comment:
`text_linkpromptbutton` seems improperly named.

Reply:
A while ago we updated the pokecrystal text command names, changing some waitbutton/promptbutton/etc ones to this current scheme. It does behave differently for link vs non-link contexts, but I'm open to a better name.

Previous discussion: https://github.com/pret/pokecrystal/issues/575
(and then https://github.com/pret/pokecrystal/commit/4aeab6b87901a3571b105564576ac356ca527172 made it `linkpromptbutton`)

Reply:
https://github.com/pret/pokered/issues/260